src/components/location/LocationStyle.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components"
```

Comment:
you can simply name the style files, `styles.js`, and the main component file to `index.js`
so there will not be a long component and style name, because we see that we are at `location` folder, we already know that this index.js and styles.js are here to be used as location component, then using another `location` text in the file name is a kind of overhead.